reporter/src/funTest/assets/evaluated-model-reporter-test-expected-output.json

```diff
@@ -270,20 +270,6 @@
       "start_line" : 1,
       "end_line" : 1,
       "scan_result" : 0
```

Comment:
Any idea why entries for the file locations 'analyzer/src/funTest/assets/projects/synthetic/gradle/lib/src/code.cpp' get removed?

Reply:
There's still one location left in that file, and I believe the other locations get consolidated into that one, as it's now not 3 separate license findings anymore, but one compound license expression finding.